server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -325,7 +332,10 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 		const ephemeralDocumentTTLSec = config.get("storage:ephemeralDocumentTTLSec") as
 			| number
 			| undefined;
-		const opsCollection = await databaseManager.getDeltaCollection(undefined, undefined);
+		const opsCollection = await databaseManager.getDeltaCollection(
+			null as unknown as string,
```

Comment:
This seems like another place where we should update the parameter typing to explicitly allow null, rather than casting.